### PR TITLE
tests: Add stability test using ctr

### LINKED ISSUE #3083: Modify stability test to use ctr for kata 2.0

### PR DESCRIPTION
**Pull request by GabyCT**
This modifies the stability test to use ctr for kata 2.0

Fixes #3083

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/stability/soak_parallel_rm.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 #
-# Copyright (c) 2017-2018 Intel Corporation
+# Copyright (c) 2020 Intel Corporation
```
Comment (liubin) [inline integration/stability/soak_parallel_rm.sh:3]: I'm suggest using `2017-2018, 2020`.

**Review by liubin — APPROVED** [review]
Thanks @GabyCT

File: integration/stability/soak_parallel_rm.sh (hunk 1 above)
Comment (fidencio) [reply integration/stability/soak_parallel_rm.sh:3]: +1 for @liubin's suggestion.

File: Makefile [hunk 2]
```diff
@@ -165,6 +165,10 @@ swarm:
 	cd integration/swarm && \
 	bats swarm.bats
 
+stability:
+	cd integration/stability && \
+	export ITERATIONS=2 && export MAX_CONTAINERS=20 && ./soak_parallel_rm.sh
```
Comment (devimc) [inline Makefile:170]: `ITERATIONS=2  MAX_CONTAINERS=20 ./soak_parallel_rm.sh`

**Review by devimc — APPROVED** [review]
thanks @GabyCT - lgtm just one nit

File: Makefile (hunk 2 above)
Comment (GabyCT) [reply Makefile]: @devimc thanks changes applied

**Comment by fidencio** [comment]
* **jenkins-ci-ubuntu-18-04**:
```
18:30:32 ctr: failed to dial "/run/containerd/containerd.sock": context deadline exceeded
18:30:32 Enabling netmon at the configuration file
18:30:32 Not a Kata runtime, not checking for Kata components
18:30:42 ctr: failed to dial "/run/containerd/containerd.sock": context deadline exceeded
18:30:42 ERROR: Unable to retry docker image docker.io/library/nginx:1.15-alpine
```

It does seem related to the docker hub pull limit.
I wonder whether it'd make sense for us to have a machine always on, caching all the images that we use, then we'd only be pulling it those from this local registry.

**Review by jodh-intel — APPROVED** [review]

**Pull request merged**